bdd/spec/018_closure_spec.sh

```diff
@@ -55,7 +55,7 @@ Describe "Closure Functions"
 
         on start {
           const numbers = [1, 2, 3, 4, 5]
-          numbers.map(double).map(fn (x: int64): string = x.toString()).join(', ').print()
```

Comment:
I am *so* happy about being able to finally eliminate these weird wrapper functions!

Reply:
amazing! :D